src/diffusers/models/unet_2d_condition.py

```diff
@@ -148,6 +153,8 @@ def __init__(
         conv_in_kernel: int = 3,
         conv_out_kernel: int = 3,
         projection_class_embeddings_input_dim: Optional[int] = None,
+        extra_film_condition_dim: int = None,
+        extra_film_use_concat: bool = False,
```

Comment:
If I'm reading it correctly, I think instead of adding these two configs, we can use the existing class embeddings config with an additional `concat` option for all the class embeddings. 

https://github.com/sanchit-gandhi/diffusers/pull/1

Here's a diff showing what I mean. I just typed it out and didn't test it but lmk if it makes sense

Reply:
Very good point! Indeed we can, resolved in https://github.com/huggingface/diffusers/pull/2232/commits/cccf556d414b5866af1b6c94f4cb396749c8d4c7. Tested in https://github.com/huggingface/diffusers/pull/2232/commits/074f883774743e2ac914704ff350c632f0f9914f.